code/modules/clothing/suits/labcoat.dm

```diff
@@ -60,19 +60,18 @@
 	update_clothing_icon()
 
 /obj/item/clothing/suit/storage/labcoat/cmo
-	name = "chief medical officer's labcoat"
+	name = "Chief Medical Officer's labcoat"
```

Comment:
why are you upper casing this? byond automatically adds the as needed (ie eats the labcoat vs eats Labcoat)

Reply:
i have a disease, it's called an english degree